### ISSUE TITLE
Consider making window/tab IDs start at 1

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#4529]
#3581 added a count for `:tab-give`, but since counts start at 1 (but window IDs start at 0), the count is offset by one, which is confusing. I guess we could make the IDs start at 1 instead?

**Comment by JadElClemens** [comment]
It is indeed confusing. The window numbers given in the :tab-give dialog appear to be 1 less than the (prefix) argument required to give a tab to the window, i.e. :tab-give 22 and '23gD' (in the default bindings) represent the same operation.

**Comment by jgkamat** [comment]
> It would be useful if there were an option to always number windows in the
> title for such operations. Perhaps there's an option for this already, but I
> didn't notice one.

You should be able to get this by adding '{id}' to window.title_format

**Comment by JadElClemens** [comment]
> You should be able to get this by adding '{id}' to window.title_format

Indeed! I thought for some reason that since the options were the same as tab.title.format that {id} would refer to the (currently active) tab id. Thanks!

**Comment by hseg** [comment]
Just got bitten by this as well - though would prefer instead to have the count logic be changed instead of the window ids be renumbered. This should be easy -- just a matter of removing the `- 1` in [tab-give](https://github.com/qutebrowser/qutebrowser/blob/268871ea860eb81fdd98090c60a0e001f55c06c0/qutebrowser/browser/commands.py#L451). Unless there's something preventing a count of 0?

**Comment by bodograumann** [comment]
Given that window ids are not necessarily consecutive, i.e. don’t correspond to indexes of some list, I think simply starting from `1` instead of `0` should be fine.

**Comment by The-Compiler** [comment]
@bodograumann I assume at the very least, some tests will need adjustments as well. If you want to work on this and ensure all tests are adjusted and working as well, a PR would certainly be welcome!

**Comment by portalgun** [comment]
Not ideal, but a quick way to get this to work is by removing the `- 1` as suggested and adding the binding `:bind tab-give g0D`
Unfortunately you can't use '0gD,' but 'g0D' seems to work just fine.
